Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
```

Comment:
We should specify package versions like we had in `requirements.txt`

Reply:
Updated with versions specified - allows for minor updates only.